pandas/tests/io/test_pyperclip.py

```diff
@@ -0,0 +1,189 @@
+# coding: utf-8
+import os
+import platform
+import random
+import string
+import unittest
+
+from pandas.io.clipboard import (
+    HAS_DISPLAY,
+    PyperclipException,
+    _executable_exists,
+    init_dev_clipboard_clipboard,
+    init_klipper_clipboard,
+    init_no_clipboard,
+    init_osx_pbcopy_clipboard,
+    init_osx_pyobjc_clipboard,
+    init_qt_clipboard,
+    init_windows_clipboard,
+    init_wsl_clipboard,
+    init_xclip_clipboard,
+    init_xsel_clipboard,
+)
+
+# import sys
+# sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..'))
+
+
+random.seed(42)  # Make the "random" tests reproducible.
+
+
+class _TestClipboard(unittest.TestCase):
```

Comment:
Hmm we don't use unittest so would need to change these - can you see how much effort it is to convert to pytest idiom?

Trying to balance asking a lot here versus getting this to work; unfortunately our current tests for this are suspect